dump/config.json

```diff
@@ -0,0 +1,870 @@
+{
```

Comment:
That was the idea, leave them here for now and decide where to move them later.
The mappings wiki is the best candidate but not clear how we could structure the mapping pages.
i.e. one mapping per page is a big overhead but a single page is too short as well